# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/opencl-utils/hotfix-sys-opencl.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- OpenCL-SDK/CMakeLists.txt.vanilla	2025-04-10 18:06:02.602687286 +0200
+++ OpenCL-SDK/CMakeLists.txt	2025-04-10 18:07:29.675247311 +0200
@@ -44,9 +44,9 @@
     set(CMAKE_INSTALL_PREFIX "${PROJECT_SOURCE_DIR}/install" CACHE PATH "Install Path" FORCE)
 endif()
 
-add_subdirectory(external/OpenCL-Headers)
-add_subdirectory(external/OpenCL-ICD-Loader)
-add_subdirectory(external/OpenCL-CLHPP)
+find_package(OpenCLHeaders)
+find_package(OpenCL)
+find_package(OpenCLHeadersCpp)
 
 list(APPEND CMAKE_MODULE_PATH
   ${PROJECT_SOURCE_DIR}/cmake
